### ISSUE TITLE
Clarify need for MAX_RECIPES

### ISSUE THREAD

**Issue by yuki-zmstr** [AY2324S2-CS2113-W14-2/tp#79]
<img width="462" alt="Screenshot 2024-03-29 at 9 39 17 PM" src="https://github.com/AY2324S2-CS2113-W14-2/tp/assets/72074553/33a34f21-6a28-4d04-acc1-94e8ce72bf71">

**Comment by yuki-zmstr** [comment]
I understand that ArrayLists are resizable arrays. Do we need this check?

**Comment by nidhi-nayak** [comment]
yep, i agree. i added that in originally because i saw prof's addressbook do something similar with an assert statement, but since arrayList's resize, i guess it is not needed! just took it out and it should reflect in the next PR.
